doc/sphinx/src/models.rst

```diff
@@ -3,3 +3,409 @@
+| :math:`T(\rho, E)`       | Temperature          | Density, Energy        |
++--------------------------+----------------------+                        |
+| :math:`P(\rho, E)`       | Pressure             |                        |
++--------------------------+----------------------+------------------------+
+| :math:`E(\rho, T)`       | Energy               | Density, Temperature   |
++--------------------------+----------------------+                        |
+| :math:`P(\rho, T)`       | Pressure             |                        |
++--------------------------+----------------------+------------------------+
+| :math:`\rho(P, T)`       | Density              | Pressure, Temperature  |
++--------------------------+----------------------+                        |
+| :math:`E(P, T)`          | Energy               |                        |
++--------------------------+----------------------+------------------------+
+| :math:`C_V(\rho, T)`     | Constant Volume      | Density, Temperature   |
++--------------------------+ Specific Heat        +------------------------+
+| :math:`C_V(\rho, E)`     | Capacity             | Density, Energy        |
++--------------------------+----------------------+------------------------+
+| :math:`B_S(\rho, T)`     | Isentropic Bulk      | Density, Temperature   |
++--------------------------+ Modulus              +------------------------+
+| :math:`B_S(\rho, E)`     |                      | Density, Energy        |
++--------------------------+----------------------+------------------------+
+| :math:`\Gamma(\rho, T)`  | Gruneisen Parameter  | Density, Temperature   |
++--------------------------+                      +------------------------+
+| :math:`\Gamma(\rho, E)`  |                      | Density, Energy        |
++--------------------------+----------------------+------------------------+
+
+A point of note is that "specific" implies that the quantity is intensive on a
+per unit mass basis.
+
+Units and conversions
+---------------------
+
+The default units for ``singularity-eos`` are cgs which results in the following
+units for thermodynamic quantities:
+
++------------+------------------+---------------------------------------+-----------------------+
+|Quantity    | Default Units    | cgµ conversion                        | Sesame Units          |
++============+==================+=======================================+=======================+
+|:math:`P`   | µbar             | 10\ :sup:`-12` Mbar                   | 10\ :sup:`-10` GPa    |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`\rho`| g/cm\ :sup:`3`   | 1                                     | 1                     |
++------------+------------------+---------------------------------------+-----------------------+
+|:math:`E`   | erg              | 10\ :sup:`-12` Mbar-cm\ :sup:`3`/g    | 10\ :sup:`-10` kJ/g   |
```

Comment:
See above comment about energy. I suggest we use \varepsilon for specific internal energy here, which has units of ergs/g.

Reply:
Yes it was a mistake for energy to be extensive. That has been fixed.